packages/features/webhooks/components/WebhookTestDisclosure.tsx

```diff
@@ -6,7 +6,7 @@ import { Badge, Button, showToast } from "@calcom/ui";
 import { Activity } from "@calcom/ui/components/icon";
 
 export default function WebhookTestDisclosure() {
-  const subscriberUrl: string = useWatch({ name: "subscriberUrl" });
+  const [subscriberUrl, webhookSecret]: [string, string] = useWatch({ name: ["subscriberUrl", "secret"] });
```

Comment:
Instead of declaring another `useWatch` for the secret, since both properties have the same type defined, it uses a single call of the react-hook-form api.